server/src/com/cloud/consoleproxy/ConsoleProxyManagerImpl.java

```diff
@@ -1425,19 +1423,39 @@ public boolean finalizeCommandsOnStart(Commands cmds, VirtualMachineProfile prof
         CheckSshCommand check = new CheckSshCommand(profile.getInstanceName(), controlNic.getIp4Address(), 3922);
         cmds.addCommand("checkSsh", check);
 
+        try {
+            File uiFiles = new File("systemvm/js");
+            if (!uiFiles.exists()) {
+                uiFiles = new File("/usr/share/cloudstack-common/systemvm/js");
+            }
+            if (uiFiles.exists() && uiFiles.isDirectory()) {
+                CopyFileInVmCommand copyFile = new CopyFileInVmCommand(uiFiles.getCanonicalPath(), "/usr/local/cloud/systemvm/js", controlNic.getIp4Address());
+                cmds.addCommand("copyFile", copyFile);
+            } else {
+                s_logger.error("Couldn't locate localization files for console proxy");
+                return false;
+            }
+        } catch (IOException e) {
+            s_logger.error("Failed to copy localization files for console proxy: " + e.getMessage());
```

Comment:
We are copying some files (using pom.xml) in that folder so these files are expected there. If these files do not exist there then he may have forgotten to add there those file. Returning false will make him identify the issue because of that.